extensions/integration-tests/src/objectExplorer.test.ts

```diff
@@ -87,7 +87,7 @@ class ObjectExplorerTester {
 		let expectedActions: string[];
 		// Properties comes from the admin-tool-ext-win extension which is for Windows only, so the item won't show up on non-Win32 platforms
 		if (process.platform === 'win32') {
-			expectedActions = ['Manage', 'New Query', 'New Notebook', 'Disconnect', 'Delete Connection', 'Refresh', 'Data-tier Application wizard', 'Launch Profiler', 'Properties'];
```

Comment:
why are we removing these 3 items here?

Reply:
Our shipped product doesn't have those extensions, so this test was failing.

Reply:
please make the else branch consistent with windows for now.